app/controllers/map.ts

```diff
@@ -6,24 +6,36 @@ import { tracked } from '@glimmer/tracking';
 import * as d3 from 'd3';
 import { geoMercator, geoPath } from 'd3-geo';
 import MapRoute from 'frontend-burgernabije-besluitendatabank/routes/map';
-import * as topojson from 'topojson';
+import { Feature, FeatureCollection } from 'geojson';
```

Comment:
No blocker for this PR, but I know that "type-only" imports are a thing in TS and I think it might make more clear that something is only imported for typing reasons. https://typescript-eslint.io/rules/consistent-type-imports/

Maybe this is something we could enable in the future if everyone agrees?

Reply:
Will make a message in rocketchat about this!